doc/sphinx-guides/source/admin/external-tools.rst

```diff
@@ -56,6 +62,9 @@ To show one of the external tools that are available in a Dataverse installation
   export TOOL_ID=1
   curl http://localhost:8080/api/admin/externalTools/$TOOL_ID
 
+  This API is Superuser only. Note the endpoint difference (/api/externalTools instead of /api/admin/externalTools).
+  curl -s -H "X-Dataverse-key:$API_TOKEN" http://localhost:8080/api/externalTools/$TOOL_ID
```

Comment:
The idea with these grey boxes is that everything inside them are commands you can execute. This breaks down when we add text into them:

![Screenshot 2024-12-16 at 2 56 51 PM](https://github.com/user-attachments/assets/1edbb487-45bd-4a2c-9db4-5c48f4a70bd1)

@stevenwinship maybe we should have two grey boxes for each command? And put the descriptive text above the second grey box?

If you want, I could bash on your branch. Or make a PR. Lemme know.

Reply:
Should be fixed now

Reply:
yes. I fixed them

Reply:
![image](https://github.com/user-attachments/assets/2c1f4534-6d96-40ec-a4bc-ba17bed5a8d9)

Reply:
Much better! Thanks!